### PR TITLE
Makes syndicate borgs explode self destruct like emagged borgs

### LINKED ISSUE #11578: Syndicate cyborgs do not get the explosion radius of bad borgs
## Test Merges: 
Not Supplied / None
## Reproduction:
Normal syndiborg self destruct is the normal small one rather then the big one you get from being emagged
 Submitted by: Maxim (181876893178527744)

### PR DESCRIPTION
**Pull request by adamsong**
# General Documentation

### Intent of your Pull Request

Make it so that syndicate borgs self destruct with the same intensity of emagged borgs

### Why is this change good for the game?
People who play syndicate borg will self destruct expecting big boom and get small boom, this removes sad boom from syndicate borg players.

# Changelog
Fixes #11578 

:cl:  
tweak: Makes the syndicate borgs self destruct like emagged borgs
/:cl:

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -608,6 +610,7 @@
 	moduleselect_icon = "malf"
 	can_be_pushed = FALSE
 	hat_offset = 3
+	syndicate_module = TRUE
```
Comment (Bibby0110) [inline code/modules/mob/living/silicon/robot/robot_modules.dm:613]: Proper autodoc please

Comment (Bibby0110) [reply code/modules/mob/living/silicon/robot/robot_modules.dm:613]: Wrong line, in the declaration of course, my bad

Comment (adamsong) [reply code/modules/mob/living/silicon/robot/robot_modules.dm:613]: What it the format for auto doc comments?

Comment (JamieD1) [reply code/modules/mob/living/silicon/robot/robot_modules.dm:613]: Add another forward slash so its /// if you could add it for the rest of the file you would make our lives easier.

Comment (adamsong) [reply code/modules/mob/living/silicon/robot/robot_modules.dm:613]: And after the variable is fine? Or should I move the comments to on top of the variable?

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**